src/service/local.proto

```diff
@@ -35,14 +35,15 @@ message height_res {
 message region_req {}
 message region_res { int32 region = 1; }
 
+enum staking_mode {
```

Comment:
Maybe overly cautious, but rename to `gateway_staking_mode` since staking is a common concept